codes/quantum/qudits/HaydenMay.yml

```diff
@@ -0,0 +1,31 @@
+#######################################################
+## This is a code entry in the error correction zoo. ##
+##       https://github.com/errorcorrectionzoo       ##
+#######################################################
+
+code_id: hayden_may
+physical: qudits
+logical: qudits
+
+name: 'Hayden May, Summoring information in spacetime'
+introduced: '\cite{arxiv:1210.0913}'
+
+description: '|
+It is well-known that quantum information, which is usually presented as a state of a qubit
+(or more generally a qudit), cannot be cloned. However, one could
+use quantum error correction schemes to store the quantum information
+in a larger domain of space, or one can store quantum information
+in long-range correlations of a many-body system. Furthermore, it is (in principle)
+very easy to clone the quantum information through time by merely waiting.
+Having said that, the paper fully characterizes what regions of spacetime
+can all hold the same quantum information by making use of the \textit{summonding}, that had been
+studeid before in the context of relativistic quantum theory by Kent.\\
+We can think of the summonding task by considering \(n\) pair of points in the spacetime,
+which are referred to as \textit{call-reveal} points, and are denoted by \( (y_i,z_i)_{i=0}^{n-1} \).
+The rule is that only one of the \textit{call points} receive a request, say \(y_j\),
+and moreover, the particle should be measured at the corresponding \textit{reveal point} \(z_j\).
+If the summoning is possible (with some predetermined strategy), then it is the case that
+the quantum information could be propagated through the region of spacetime determined by the call-reveal points.\\
+Authors make use of quantum error correction schemes to prove that the fundumental laws of relativity and quantum information,
+which are causality and no-cloning, are the only conditions that determine the feasibility of a summoning task (obviously these conditions are necessary,
+but the non-trivial part is the sufficiency; this is where quantum error correction techniques are needed).'

```

Comment:
Missing parent and cousin entries. For parent, what is the smallest code family with the same alphabet (qudits)? For cousins, please add niset_andersen_cerf and homological_cv, with appropriate descriptions as to why they are cousins.